src/cljc/matcher_combinators/matchers.cljc

```diff
@@ -31,6 +31,19 @@
         :else
         (core/->Value expected)))
 
+(declare match-with)
+
+(defn deep-equals
```

Comment:
❓  question (non_blocking): Do you think we should create a test for it? It would be very similar to the `match-with` [test](https://github.com/nubank/matcher-combinators/blob/ccd20994884733860b7995f418b065bfee3c3c55/test/clj/matcher_combinators/matchers_test.clj#L260)

Reply:
I would be in favor of a test. Even if it is a little redundant, if nothing else it serves as additional documentation.